src/main/res/values/styles.xml

```diff
@@ -73,4 +73,20 @@
         <item name="android:radius">@dimen/flat_button_corner_radius</item>
     </style>
 
+    <style name="CompassShowcaseView" parent="ShowcaseView">
+        <item name="sv_backgroundColor">#AA607d8b</item>
```

Comment:
I found the tour to be hard to read, and the overlay was perhaps a bit _too_ transparent. Can we use `#EE607d8b` for the `sv_backgroundColor`?